Description: Change the default PATH
 The configured default PATH is used in the parts that run as root, the
 greeter and for the user  sessions, so its better to have an spartan default.
 .
 At the same time, for the zsh users this spartan version gets expanded thanks
 to the default global zshenv, while exporting a different value in the
 Xsession script would be overriding the users configurations in their local
 zshenv.
Author: Maximiliano Curia <maxy@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-14
---
Index: sddm/data/man/sddm.conf.rst.in
===================================================================
--- sddm.orig/data/man/sddm.conf.rst.in	2015-09-14 18:15:17.000000000 +0200
+++ sddm/data/man/sddm.conf.rst.in	2015-09-14 19:09:48.946429964 +0200
@@ -114,7 +114,7 @@
 
 `DefaultPath=`
 	Default path to set after successfully logging in.
-	Default value is "/bin:/usr/bin:/usr/local/bin".
+	Default value is "/bin:/usr/bin".
 
 `MinimumUid=`
 	Minimum user id of the users to be listed in the
Index: sddm/src/common/Configuration.h
===================================================================
--- sddm.orig/src/common/Configuration.h	2015-09-14 19:04:20.239415239 +0200
+++ sddm/src/common/Configuration.h	2015-09-14 19:09:40.058781080 +0200
@@ -73,7 +73,7 @@
                                                                                                           "A script to execute when starting the desktop session"));
         );
         Section(Users,
-            Entry(DefaultPath,         QString,     _S("/bin:/usr/bin:/usr/local/bin"),         _S("Default $PATH"));
+            Entry(DefaultPath,         QString,     _S("/bin:/usr/bin"),                        _S("Default $PATH"));
             Entry(MinimumUid,          int,         UID_MIN,                                    _S("Minimum user id for displayed users"));
             Entry(MaximumUid,          int,         UID_MAX,                                    _S("Maximum user id for displayed users"));
             Entry(HideUsers,           QStringList, QStringList(),                              _S("Hidden users"));
